modules/adxpremiumAnalyticsAdapter.js

```diff
@@ -5,7 +5,9 @@ import CONSTANTS from '../src/constants.json';
 import * as utils from '../src/utils.js';
 
 const analyticsType = 'endpoint';
-const url = 'https://adxpremium.services/graphql';
+const defaultUrl = 'https://adxpremium.services/graphql';
+
+let reqCountry = reqCountry || null;
```

Comment:
hey @adxpremium , there's a lint error here:

```
'reqCountry' was used before it was defined.eslint(no-use-before-define)
```
Is `reqCountry` expected to be on the window object?

Reply:
Hey @aleksatr, yes, reqCountry is expected to be on the window object, if not then it will be just null and that's fine.

Reply:
Ok, in that case, I guess you can do something like:

```
let reqCountry = window.reqCountry || null;
```

to mitigate the lint error.

Reply:
Ok thanks, I will update in a moment.